add audio/m4a and audio/x-m4a mime types to list of handled MIME types (Closes: #490402).

Patch from upstream, see
<URL:http://sourceforge.net/mailarchive/message.php?msg_name=E1KHmRH-0000m1-4R%40mail.sourceforge.net>.

--- mplayerplug-in-3.55.orig/Source/plugin-setup.cpp
+++ mplayerplug-in-3.55/Source/plugin-setup.cpp
@@ -348,6 +348,8 @@
 		    "audio/x-mpeg:mpg,mpeg:MPEG;"
 		    "audio/mpeg2:mp2:MPEG audio;"
 		    "audio/x-mpeg2:mp2:MPEG audio;"
+		    "audio/mp4:mp4:MPEG 4 audio;"
+		    "audio/x-mp4:mp4:MPEG 4 audio;"
 		    "video/mp4:mp4:MPEG 4 Video;"
 		    "video/3gpp:mp4,3gp:MPEG 4 Video;", sizeof(MimeTypes));
 
